src/python/pants/task/console_task.py

```diff
@@ -34,10 +25,6 @@ def register_options(cls, register):
     register('--output-file', metavar='<path>',
              help='Write the console output to this file instead.')
 
-    if cls._register_console_transitivity_option:
```

Comment:
The TODO in `jvm_platform_analysis.py` was really nonspecific, I'm super sorry about that. I also think that TODO is actually incorrect.

The *previous* intention was for `--transitive` to be registered for *all* `ConsoleTask`s, which here would just mean removing the `if`.

**However**, I think that TODO is actually wrong, and that `_register_console_transitivity_option` should stay unchanged, and then **the only change needed in this diff** should be the changes in `jvm_platform_analysis.py`, I believe.

Reply:
No problem - I've reverted the file but deleted the comment for posterity.